kibana.md

```diff
@@ -157,6 +156,35 @@ When the ES plugin boots up, it follows some logic to evaluate where to read the
 
 ![config loading diagram](ror_config_loading_diagram.png)
 
+### Malformed configuration index 
```

Comment:
We decided long ago to always use the word "settings" to indicate the ROR YAML, and "in-index settings" to indicate the YAML stored in .readonlyrest